cluster/gce/configure.sh

```diff
@@ -36,9 +36,6 @@ fetch_metadata() {
   fi
 }
 
-# DEPLOY_PATH is the gcs path where cri-containerd tarball is stored.
-DEPLOY_PATH=${DEPLOY_PATH:-"cri-containerd-release"}
-
```

Comment:
nit: https://storage.googleapis.com/cri-containerd-release is uses a couple times might want to add it back?

Reply:
OK. Will change it to a constant.

Reply:
Done.